HW_10.py/task_001.py

```diff
@@ -0,0 +1,17 @@
+import random
+import pandas as pd
+import numpy as np
+
+lst = ['robot'] * 10
+lst += ['human'] * 10
+random.shuffle(lst)
+data = pd.DataFrame({'whoAmI': lst})
+data = pd.get_dummies(data['whoAmI'])
+
+print(f'one hot coding with get_dummies:\n{data}\n')
+
+data = pd.DataFrame(0, index=np.arange(len(lst)), columns=sorted(list(set(lst))))
```

Comment:
У вас уже есть переменная `data`, ее не нужно заново переопределять. Вместо этого можно сделать следующее
```suggestion
data[list(set(data.whoAmI))] = 0
```